### DISCUSSION TITLE
Nag Screen Is Very Poorly Implemented

### DISCUSSION THREAD

**Discussion by jaypatrick** [community/community#33990] [Copilot Conversations]
The nag screen that pops up every single time VS is started is ridiculous as it steals focus, and why does it pop up every single time VS is opened? There is no option to permanently silence it, and it is interruptive to our team, so we are instructing our devs to uninstall it. Give us an option to silence the nag screen as it serves no real purpose other than to interrupt workflows and annoy our devs.

**Comment by iamcjwilliams** [comment]
Hello, @jaypatrick Our copilot team is looking into this issue

**Comment by tilmann1** [comment]
@jaypatrick Can you please share a screenshot of the nag screen that you're seeing? There should be no nag screen when you're on a subscription.
